test-suite/shared/src/test/scala/org/scalajs/testsuite/niocharset/BaseCharsetTest.scala

```diff
@@ -14,14 +14,13 @@ import scala.util._
 import java.nio._
 import java.nio.charset._
 
-import scala.scalajs.js
-import js.JSConverters._
+import org.junit.Assert._
 
-import org.scalajs.jasminetest.JasmineTest
-
-class BaseCharsetTest(val charset: Charset) extends JasmineTest {
+class BaseCharsetTest(val charset: Charset) {
   import BaseCharsetTest._
 
+  protected def inJVM: Boolean = 1.0.toString != "1"
```

Comment:
I will do that. 
This should only be used in cases where there is a bug found during the migration of tests. I would assume that once those bugs are fixed we should remove this utility and never use it again.